src/compile-cache.js

```diff
@@ -1,5 +1,10 @@
 'use strict'
 
+// For now, we're not using babel or ES6 features like `let` and `const` in
+// this file, because `apm` requires this file directly in order to pre-warm
+// Atom's compile-cache when installing or updating packages, using an older
+// version of node.js
```

Comment:
@mnquintana There's an open PR to upgrade apm's node version to 4.x, but there are some issues with compatibility with npm. I forget the details. We definitely would like to upgrade it soon.